The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2025-02-14 13:11:29.231 UTC [46] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:11:29.272 UTC [47] LOG:  database system was shut down at 2025-02-14 13:11:29 UTC
2025-02-14 13:11:29.279 UTC [46] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down...2025-02-14 13:11:29.583 UTC [46] LOG:  received fast shutdown request
.2025-02-14 13:11:29.584 UTC [46] LOG:  aborting any active transactions
2025-02-14 13:11:29.585 UTC [46] LOG:  background worker "logical replication launcher" (PID 53) exited with exit code 1
2025-02-14 13:11:29.585 UTC [48] LOG:  shutting down
2025-02-14 13:11:29.592 UTC [46] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 13:11:29.691 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 13:11:29.691 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 13:11:29.693 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:11:29.699 UTC [64] LOG:  database system was shut down at 2025-02-14 13:11:29 UTC
2025-02-14 13:11:29.703 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 13:11:36.097 UTC [79] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 13:11:36.097 UTC [79] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 13:11:36.410 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:11:36.410 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_e509b6ba-0ac3-4131-b698-9c73746ca9f2" WITH (FORCE)
2025-02-14 13:11:37.002 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:11:37.002 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_21e67df2-4a67-4eda-b049-85baedba0811" WITH (FORCE)
2025-02-14 13:11:37.507 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:11:37.507 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_644aacbd-2881-4a88-8f3a-a134bc902f1b" WITH (FORCE)
2025-02-14 13:11:38.018 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:11:38.018 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_b02a6d1e-690c-410f-8c90-77f31bd572d9" WITH (FORCE)
